doc/mboxgrep.1

```diff
@@ -79,6 +79,11 @@ Pipe each found message to COMMAND
 .IP "-d, --delete"
 Suppress normal output and delete selected messages instead.  Use with
 caution.
+.IP "-Y, --passthrough"
+Write matches to file, pipe to stdout the rest. Useful to split a file 
```

Comment:
I see this as an unnecessary feature. The same result can be achieved by invoking mboxgrep once with '-v' and once without.

I don't want unnecessary feature complexity.